### ISSUE TITLE
Timezone id could not be parsed

### ISSUE THREAD

**Issue by EHJ-52n** [52North/WeatherDataCollector#4]
The current implementation to parse the timezone is failing.

Log output
```
2015-05-11 00:09:58,578 INFO [52n-wdc-timer] [MuensterwetterRealTimeCollector.java:372] Timezone id '(UTC+1)
' is not supported by Java. Please check 'http://docs.oracle.com/javase/7/docs/api/java/util/TimeZone.html'.
```

It is a bit related to #3.

**Comment by nuest** [comment]
Could it be the case that the data you're downloading now actually refers to the new Weather station (HEI2?) ? Should we get in touch? Because that would explain why the timestamp format changed.

**Comment by EHJ-52n** [comment]
@nuest The problem was that they used the timezone identifier "UTC+1" which must be translated to "Etc/GMT-1".

From my point of view this issue is fixed by d97435f9fea0fa9acc.

In addition, we might get in touch to check, if I am using the correct data file.
